### PR TITLE
csp bugfix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25454

## Description
Breaking BC, returning the arg in a list.

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 1]
```diff
@@ -460,12 +460,15 @@ def test_argToList():
     test2 = 'a,b,c'
     test3 = '["a","b","c"]'
     test4 = 'a;b;c'
+    test5 = 1
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:463]: add also
```suggestion
    test5 = 1
    test6 = '1'
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 2]
```diff
@@ -461,13 +461,17 @@ def test_argToList():
     test3 = '["a","b","c"]'
     test4 = 'a;b;c'
     test5 = 1
+    test6 = '1'
+    test7 = True
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:465]: 💯

File: Packs/Base/ReleaseNotes/1_0_7.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+- __CommonServerPython__
+Fixed an issue where the **argToList** function did not behave as expected. This fix is breaking backward computability.
```
Comment (fvigo) [inline Packs/Base/ReleaseNotes/1_0_7.md:4]:
```suggestion
Fixed an issue where the **argToList** function did not behave as expected. This fix is breaking backward compatibility.
```

File: Packs/Base/Scripts/CommonServerPython/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ## [Unreleased]
   - Added **Endpoint** Common class.
   - Added a new function **auto_detect_indicator_type** which detects indicators. 
+  - Fixed an issue where the **argToList** function did not behave as expected. This fix is breaking backward computability.
```
Comment (fvigo) [inline Packs/Base/Scripts/CommonServerPython/CHANGELOG.md:4]:
```suggestion
  - Fixed an issue where the **argToList** function did not behave as expected. This fix is breaking backward compatibility.
```

**Comment by kirbles19** [comment]
@guyfreund Done.

**Comment by ShahafBenYakir** [comment]
Falling on MP step (non-related) force merging

**Pull request merged**